app/assets/stylesheets/home/components/_variables.scss

```diff
@@ -1,6 +1,19 @@
 $text-color: #0077B6;
 $breakpoint-desktop: 992px;
 
+@mixin flex-column{
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
If you don't need params it would be a better approach to use the [Extend/Inheritance
](https://sass-lang.com/guide) feature

Reply:
Okay! 
I'm going to change it!